src/renderer/components/Preferences/Account.vue

```diff
@@ -130,32 +131,38 @@ export default {
 
 <style lang="scss" scoped>
 #account {
-  .el-table /deep/ {
-    tr,
-    th,
-    td {
-      background-color: var(--theme-background-color);
-      color: var(--theme-secondary-color);
-      border-bottom: 1px solid var(--theme-border-color);
+  .connected-account {
+    .el-table /deep/ {
+      tr,
+      th,
+      td {
+        background-color: var(--theme-background-color);
+        color: var(--theme-primary-color);
+        border-bottom: 1px solid var(--theme-border-color);
+      }
     }
 
-  }
+    .el-table::before {
+      background-color: var(--theme-border-color);
+    }
 
-  .el-table::before {
-    background-color: var(--theme-border-color);
-  }
+    .allow-up,
+    .allow-down {
+      display: inline-block;
+    }
 
-  .reset {
-    margin: 24px 12px;
-    text-align: right;
+    .action {
+      font-size: var(--base-font-size);
+    }
   }
-}
 
-.allow-up {
-  padding: 0;
-}
+  .section /deep/ {
```

Comment:
__[stylelint]__ <sub>posted by Sider</sub>

Expected selector "#account .section /deep/" to come before selector "#account .connected-account .el-table /deep/" (no-descending-specificity)